Tests/SubviewDistributionTests.swift

```diff
@@ -41,13 +41,14 @@ class SubviewDistributionTests: XCTestCase {
             a <> ~1~ <> b <> ~2~ <> c
         }
         
-        let relative1Space = a.frame.minY
+        // Space before a is implicity ~1~.
+        let relative1Space = (a.top |--| .top).height
         
         let pixelRounder = PixelRounder(for: view)
         // Assert that our spacing is correct above.
-        XCTAssertEqualWithAccuracy(b.frame.minY - a.frame.maxY, relative1Space, accuracy: pixelRounder.pixelAccuracy)
-        XCTAssertEqualWithAccuracy(c.frame.minY - b.frame.maxY, 2 * relative1Space, accuracy: pixelRounder.pixelAccuracy)
-        XCTAssertEqualWithAccuracy((view.frame.maxY - c.frame.maxY), relative1Space, accuracy: pixelRounder.pixelAccuracy)
+        XCTAssertEqualWithAccuracy((a.bottom |--| b.top).height, relative1Space, accuracy: pixelRounder.pixelAccuracy)
```

Comment:
It would be better. But I do like having this test be as readable as it is. Also, since alignment doesn't depend on the measuring code, I think we're good to go here.